src/chrome/js/popup.js

```diff
@@ -24,7 +24,19 @@ var opts = {
 chrome.tabs.query({active: true,lastFocusedWindow: true}, function(tabs) {
   // and use that tab to fill in url and all page information
   var tab = tabs[0],
-      tabURL = tab.url;
+      tabURL = tab.url,
```

Comment:
Using this script both a full URL and a path will get a the protocol and hostname. For the full URL this results in a non-existing URL. Could you add a check and only add the protocol and host when a path is found?